eland/ml/ml_model.py

```diff
@@ -523,18 +525,23 @@ def _import_model(
         elif es_if_exists == "replace":
             ml_model.delete_model()
 
+        trained_model_input = None
+        is_ltr = next(iter(inference_config)) is TYPE_LEARNING_TO_RANK
+        if not is_ltr or es_version(es_client) < (8, 15):
```

Comment:
You should also check `or build_flavor == serverless` as serverless reports its version as 8.11. 

See #698 for an example

Reply:
This is a good catch. I will add this.

Reply:
@davidkyle I introduced a new method to check is the build flavor is serverless in commons and updated the code.
I have the intuition that it will be used in other places in the future.

I also updated the existing check in `eland/cli/eland_import_hub_model.py` to use this new method.